metadata_layer.py

```diff
@@ -1576,6 +1576,26 @@ def _key(classification):
                 # is associated with the original image.
                 self.make_thumbnail(data_source, link, link_obj)
 
+        # The metadata wrangler doesn't need information from these data sources.
+        # We don't need to send it information it originally provided, and
+        # Overdrive makes metadata accessible to everyone without buying licenses
+        # for the book, so the metadata wrangler can obtain it directly from
+        # Overdrive.
+        # TODO: Remove Bibliotheca and Axis 360 from this list.
+        METADATA_UPLOAD_BLACKLIST = [
+            DataSource.METADATA_WRANGLER,
+            DataSource.OVERDRIVE,
+            DataSource.BIBLIOTHECA,
+            DataSource.AXIS_360,
+        ]
+        if made_core_changes and (not data_source.integration_client) and (data_source.name not in METADATA_UPLOAD_BLACKLIST):
+            # Create a transient failure CoverageRecord for this edition
+            # so it will be processed by the MetadataUploadCoverageProvider.
+            internal_processing = DataSource.lookup(_db, DataSource.INTERNAL_PROCESSING)
+            CoverageRecord.add_for(edition, internal_processing,
+                                   operation=CoverageRecord.METADATA_UPLOAD_OPERATION,
+                                   status=CoverageRecord.TRANSIENT_FAILURE,
+                                   collection=collection)
```

Comment:
If there already is a successful CoverageRecord for this action, `add_for()` will switch it back from success to transient failure. I feel like this is a shortcoming of add_for() rather than a problem with the new code.

Reply:
Okay, I was thinking we wanted to re-sync for every change.

Reply:
In theory, we do, I'm just concerned about performance and I think it's better to take it slow since right now we really only _need_ this for cover images.